Change Mensa Datasource Host

henry-herrmann opened the pull request:
This PR changes the host of our mensa datasource to a server at the RUB.

domai-tb reviewed (approved):
- Where is the "web" package comming from?
- Minimal Dart Version should not be a dev version

henry-herrmann commented:
> * Where is the "web" package comming from?
> * Minimal Dart Version should not be a dev version

That definitely shouldn't be there. Removed the new package and reverted the dart version back to what it used to be.
